pipeline/backend/kubernetes/pod.go

```diff
@@ -183,7 +183,7 @@ func podContainer(step *types.Step, podName, goos string, options BackendOptions
 	container := v1.Container{
 		Name:            podName,
 		Image:           step.Image,
-		WorkingDir:      step.WorkspaceBase,
+		WorkingDir:      step.WorkspaceBase, // step.WorkingDir will be respected by the generated script // TODO: plugins wont work with this at all!!!
```

Comment:
@pat-s what should we do with plugins who explizite wana run with there own entrypoint and as such we can not `cd` via an self generated script

?

Reply:
this should have `step.WorkingDir` imho